src/ApparentHorizons/Tags.hpp

```diff
@@ -322,6 +322,28 @@ struct UnitNormalOneFormCompute : UnitNormalOneForm<Frame>, db::ComputeTag {
   using return_type = tnsr::i<DataVector, 3, Frame>;
 };
 
+/// UnitNormalVector is defined as \f$S^i = g^{ij} S_j\f$,
+/// where \f$S_j\f$ is the unit normal one form
+/// and \f$g^{ij}\f$ is the inverse spatial metric.
```

Comment:
okay, the compute item looks good; one more thing I noticed; we are using \gamma^{ij} for the inverse spatial metric, (g is the spacetime metric); you can squash everything when fixing this